networking/amplify/backend/api/mobile/schema.graphql

```diff
@@ -31,6 +31,7 @@ type GameTemplate @model{
   imageUrl: String
   questionTemplates: [QuestionTemplate] @manyToMany(relationName: "GameQuestions") 
   questionTemplatesCount: Int! @default(value: "0")
```

Comment:
required to maintain a consistent order to the questions between game creation in `central` and gameplay in `host/play` (otherwise, we are just adding them based on when the API request for question creation is fired off)